mozilla/fx-private-relay
# Expose updated plans in the API

# New feature description

This PR fixes MPP-2339. The most important work is exposing the new plan data from the API (2240b67a5a7063a4a01c0e7c8cad6dabc614550d). It then also uses the newly-exposed data to generate the proper subscription links in the frontend, and adds some helpers to access the newly-exposed data (27e8e750171345806667ccd9a016e3e061a10565). And finally, with a plan-country mapping available, I've removed phone UI bits for people in countries where it's not available (4deb476e158916381e2cf3971fd9905fc055a4eb). Oh, and I've removed a hardcoded price and replaced it by one from the API (5403e0e1).

Note that there are still some questions about some plan IDs (notably, Ireland and Austria have none yet), and final pricing is not known yet. However, at least this will give us a single place to update that.

# How to test

Nothing should have changed, except if you're in a country in which phones is not available, in which case you should no longer see the UI. Additionally, the price in the onboarding screen should now show `$?.??` instead of a hardcoded value, to indicate that that still needs to be set.

# Checklist
